demo/transient/transient_snes_mpfaof90.cfg

```diff
@@ -1,6 +1,7 @@
 [suites]
```

Comment:
At some point, it might be nice to add comments describing the problems/initial conditions described by the parameters in these files.